src/beautifiers/hh_format.coffee

```diff
@@ -0,0 +1,22 @@
+###
+Requires http://hhvm.com/
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class HhFormat extends Beautifier
+  name: "hh_format"
+  link: "http://hhvm.com/"
+
+  options:
+    PHP: true
```

Comment:
This should be `PHP: false` since the language `PHP` is supported (adding the key `PHP` accomplishes this) and *no options* are supported (indicated by value `false` instead of `true`).